.github/workflows/security.yml

```diff
@@ -0,0 +1,19 @@
+name: Perform Weekly Security Audit with ZAProxy
+# Use ZAP Proxy to perform a full scan of the production site.
+# Scan automatically opens an issue after completion
+# with results of the audit.
+
+on:
```

Comment:
It would be helpful to be able to manually trigger this job as well. We should be able to set up a `workflow_dispatch` trigger similarly to this (but without any `input`s): https://github.com/jeremyckahn/farmhand/blob/7591a456ff506f4e2e1c579b443b9097595ee7aa/.github/workflows/run-release.yml#L4-L5